File: src/bonAppetit.scala
    def bonAppetit(bill: Array[Int], k: Int, b: Int) {
        val annaTotal = (bill.patch(k, Nil, 1).sum / 2).toInt
        if (annaTotal == b) {
            println("Bon Appetit")
        } else {
            println( (b - annaTotal).toString)
        }
    }

File: src/bonAppetit.py

def bonAppetit(bill, k, b):
    bill.pop(k)
    subTotal = sum(bill)
    annaTotal = int(subTotal / 2)
    if annaTotal == b:
        print("Bon Appetit")
    else:
        print(str(b-annaTotal))

